JS-Array-Prototype-ForEach.md

```diff
@@ -52,3 +52,18 @@ numbers.forEach(
 document.write(sum);
 // Output: 33
 ```
+
+## How to create your own forEach for learning purposes
+```javascript
+function forEach(array, callback){
+  for(let i = 0; i < array.length; i++){
+    //apply the passed in callback on each element in the array
+    callback(array[i]);
+  }
+}
+function logValue(value, index){
+  console.log(`Element is: ${value}` )
```

Comment:
That's ok. I also noticed the usage of `let`.

---

I am not against using ES6. But we don't teach ES6 syntax in our core curriculum. Even, the codemirror editor we use, gives syntax error when we use arrow functions etc. from ES6.

The writer should at the very least mention that this is in ES6 syntax. Or simply use ES5 syntax, because target audience would be most familiar with it.